Src/FluentAssertions/Collections/CollectionAssertions.cs

```diff
@@ -936,9 +936,26 @@ public AndConstraint<TAssertions> BeInDescendingOrder(IComparer<object> comparer
         /// <param name="becauseArgs">
         /// Zero or more objects to format using the placeholders in <see cref="because" />.
         /// </param>
+        [Obsolete("Use NotBeInAscendingOrder instead")]
         public AndConstraint<TAssertions> NotBeAscendingInOrder(string because = "", params object[] becauseArgs)
         {
-            return NotBeAscendingInOrder(Comparer<object>.Default, because, becauseArgs);
+            return NotBeInAscendingOrder(Comparer<object>.Default, because, becauseArgs);
+        }
+
+        /// <summary>
+        /// Asserts the current collection does not have all elements in ascending order. Elements are compared
+        /// using their <see cref="IComparable.CompareTo(object)" /> implementation.
+        /// </summary>
+        /// <param name="because">
+        /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion
+        /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+        /// </param>
+        /// <param name="becauseArgs">
+        /// Zero or more objects to format using the placeholders in <see cref="because" />.
+        /// </param>
+        public AndConstraint<TAssertions> NotBeInAscendingOrder(string because = "", params object[] becauseArgs)
```

Comment:
I think we should extend the docs for all the new methods to include the "rules" for empty and single-element collections in the `<remarks>`. 

E.g., today, a colleague asked me what happens if you call `collection.Should().OnlyContain` on an empty collection. I had to look at the implementation to find the answer.